programs/helium-sub-daos/src/instructions/staking/claim_rewards_v0.rs

```diff
@@ -0,0 +1,167 @@
+use crate::{
+  current_epoch, error::ErrorCode, find_allocation_index, state::*, GetPercent, EPOCH_LENGTH,
+  TESTING,
+};
+use anchor_lang::prelude::*;
+use anchor_spl::{
+  associated_token::AssociatedToken,
+  token::{Mint, Token, TokenAccount},
+};
+use circuit_breaker::{
+  cpi::{accounts::TransferV0, transfer_v0},
+  CircuitBreaker, TransferArgsV0,
+};
+use voter_stake_registry::state::{Registrar, Voter};
+
+#[derive(AnchorSerialize, AnchorDeserialize, Clone, Default)]
+pub struct ClaimRewardsArgsV0 {
+  pub deposit_entry_idx: u8,
+  pub epoch: u64,
+}
+
+#[derive(Accounts)]
+#[instruction(args: ClaimRewardsArgsV0)]
+pub struct ClaimRewardsV0<'info> {
+  #[account(
+    mut,
+    seeds = [registrar.key().as_ref(), b"voter".as_ref(), voter_authority.key().as_ref()],
+    seeds::program = vsr_program.key(),
+    bump,
+    has_one = voter_authority,
+    has_one = registrar,
+  )]
+  pub vsr_voter: AccountLoader<'info, Voter>,
+  #[account(mut)]
+  pub voter_authority: Signer<'info>,
+  pub registrar: AccountLoader<'info, Registrar>,
+
+  #[account(
+    mut,
+    seeds = ["stake_position".as_bytes(), voter_authority.key().as_ref(), &[args.deposit_entry_idx]],
+    bump,
+  )]
+  pub stake_position: Account<'info, StakePositionV0>,
+
+  #[account(
+    mut,
+    has_one = staker_pool,
+    has_one = dnt_mint,
+  )]
+  pub sub_dao: Account<'info, SubDaoV0>,
+  pub dnt_mint: Box<Account<'info, Mint>>,
+
+  #[account(
+    seeds = ["sub_dao_epoch_info".as_bytes(), sub_dao.key().as_ref(), &args.epoch.to_le_bytes()], // Break into 30m epochs
```

Comment:
Probably have this elsewhere but epoch is 24h not 30m

Reply:
Removed all instances of this in my next pr